src/main/webapp/app/entities/text-assessments/text-assessments.service.ts

```diff
@@ -51,8 +52,10 @@ export class TextAssessmentsService {
             .map((res: EntityResponseType) => this.convertResponse(res));
     }
 
-    public getFeedbackDataForExerciseSubmission(exerciseId: number, submissionId: number): Observable<StudentParticipation> {
-        return this.http.get<StudentParticipation>(`${this.resourceUrl}/exercise/${exerciseId}/submission/${submissionId}`);
+    public getFeedbackDataForExerciseSubmission(submissionId: number): Observable<StudentParticipation> {
+        return this.http
+            .get<StudentParticipation>(`${this.resourceUrl}/submission/${submissionId}`)
+            .pipe(tap((sp: StudentParticipation) => (sp.submissions[0].result = sp.results[0])));
```

Comment:
What is this doing?

Reply:
I added a comment to explain that this will wire up the result to the submission.